analysis/structure/uconn.go

```diff
@@ -7,6 +7,26 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+// GetConnSourcesFromDest finds all of the ips which communicated with a
+// given destination ip
+func GetConnSourcesFromDest(res *database.Resources, ip string) []string {
```

Comment:
Often times we have the need to find all of the ip addresses which connected to a certain destination address.